AttestationPoint.md

```diff
@@ -0,0 +1,108 @@
+# Deterministic Attestation Point Computation
+
+## Introduction
+
+Discreet Log Contracts (DLCs) are oracle contracts enforced using a set of adaptor signature "point-locks."
+This means that in order to spend any execution branch of a DLC, one must reveal the scalar pre-image for a point.
+As is described in this document, the points used for adaptor signing correspond to anticipations of all
+possible oracle attestations such that exactly one adaptor secret is revealed, unlocking exactly one signature,
+making exactly one Contract Execution Transaction (CET) valid for on-chain publication.
+
+By an anticipation of an oracle attestation, we mean an elliptic curve point `S` whose scalar pre-image `s` (i.e. `S = s*G`)
+is an oracle attestation of a specific message, `m` (or the sum of multiple such attestations).
+The point `S` can be thought of as an encryption key to be used on the signature of the CET which corresponds to the
+event where `m` is attested to so that this signature can only be used should this attestation be broadcast.
+The key point is that `S` can be computed *in advance* given information in oracle announcements so that anticipation points
+(aka attestation points or sometimes adaptor points) can be used to construct a DLC and later the corresponding attestations
```

Comment:
I feel there is a distinction between attestation point and adaptor point, as an adaptor point can be the sum of multiple attestation points.